src/main/webapp/auth.js

```diff
@@ -0,0 +1,95 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file contains functionality relating to Google Sign in
+// and storing cookies
+// TODO: Manage User Sign In State
+
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
+
+// OAUTH 2.0 Client ID
+// TODO: Add Client ID once GCP project is made
+const CLIENT_ID = 'INPUT_CLIENT_ID_HERE';
```

Comment:
Will it be safe to have the client id in plain text?

Reply:
So from what I'm reading, it should be fine since it's a public identifier. Also worth noting that all hosts have to whitelisted to even use this client ID (so if it did get to someone with malicious intent, it's basically useless unless if they somehow whitelist themselves).